CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.3.1
+ - Fix deadlock that could occur in certain odd situations. If you noticed this plugin stalling this should fix it.
```

Comment:
can we describe this a bit more or just say it fixes a deadlock? it seems the deadlock would occur pretty much every time in any situation?